source/data/tier1data/clients/mango_portal.rst

```diff
@@ -0,0 +1,179 @@
+It allows users to manage their data in an intuitive way, without any installations,
+with a strong focus on managing :ref:`metadata<metadata>`.
+
+When logging in, you will be redirected to the login page of your institution. 
+This takes you to an overview with one or multiple zones with at least one of the following options:
+
+- Select 'Enter portal' to enter that zone via the ManGO portal.
+- Select 'How to Connect' to get credentials for logging in to other clients, like :ref:`iCommands` or the :ref:`the PRC<python-client>`.
+- Selecting the downward arrow opens an overview of all projects you are member of in that zone. Clicking on the project name sends you to the project management page. 
+
+If you select the first option, you will be sent to the ManGO Portal home page:
+
+.. figure:: ../images/mango_portal/mango_portal_main_page.png
+  :width: 1000
+  :alt: The main page of the ManGO portal
+
+
+This main page has four main tabs:
+
+- **Collections**, where you can manage the data in your collections.
+- **Search**, where you can search for data objects and collections based on different criteria.
+- **Trash**, where you can inspect and manage your :ref:`trash collection <trash>`.
+- **Metadata schemas**, where you can view and mana metadata schemas.
+
+*************************************
+Managing collections and data objects
+*************************************
+
+The Collections tab provides an overview of the collections you have access to and lets you browse through them.
+Clicking on the name of a subcollection shows you its contents under the "Content" tab: both subcollections and data objects.
+If you click on a data object instead, you go to its page, which does not have such a "Content" tab.
+
+Above the name of the current collection or data object, a breadcrumb menu allows you to go back to a collection on top of it. 
+Next to the name, clicking on the pencil icon allows you to rename the active collection or data object.
+
+The page of a collection also includes buttons to create a new collection and upload files,
+which are only available if you have the right :ref:`permissions<collaboration>`.
+Also depending on your permissions you may see a dropdown menu of bulk actions, allowing you to move, copy or delete one or more
+collections or data objects at a time.
+To do so, click on the checkbox next to the names of the collections or data objects, select your desired action and click on "apply".
+When copying or moving, a list of collections will appear where you can browse to select a destination.
+Notice that only data objects, not collections, can be copied.
+
+Data objects do not have a "Content" tab, as they cannot contain any objects. 
+Instead, they have a "System properties" tab, which contains some basic information about the object.
+They also provide a "Preview" tab in which, depending on the file type, you may be able to see the contents of your data object.
```

Comment:
It would be good to list which files types can be pre-viewed

Reply:
Applied